### PR TITLE
Change to testFailureIngnore to false and improving the time measuring closes #23

### LINKED ISSUE #23: Jenkins job allways ends with SUCCESS status
I've switched the git plugin used to trigger jenkins, and while testing it, I discovered that the jenkins job ends with success status in all cases (if a test fails or even if a test does not compile).

You can check branch `test-switch-git-plugin` and the jenkins builds used to test this behaviour:
- build 1: same source than main branch, result success as expected
- build 2: the expected string of a test changed. The test fails (as seen in the log), but jenkins returns SUCCESS (all steps are green)
- build 3: use an syntactically wrong method. The test does not compile (as seen in the log),  but jenkins returns SUCCESS (all steps are green)

### PR DESCRIPTION
**Pull request by augustocristian**
Minor changes:

- The report incorrectly showed success even when a test failed due to the `testFailureIgnore` setting being set to true. (This setting has now been changed to false.)
- The time measuring when a test fails was not correct (missing end time due to a early return of the script) now its solved.
- Exceptions correctly managed, now the  resources are always stopped without regarding the step fails or not.
- Added -DskipTests to the mvn test command in the actions build.yml to avoid test execution

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -31,27 +31,48 @@ pipeline {
...
-              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-teardown.sh tjobc 0'
+              script {
+                try{
+                  sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-setup.sh tjobc 0'
+                  sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-testexecution.sh tjobc 0 http:// 5000 "CatalogTests#addProductsToBasket"'
+                  sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-teardown.sh tjobc 0'
+                } catch (err) {
+                  sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-teardown.sh tjobc 0'
+                  throw err
+                } // EndCatch
```
Comment (javiertuya) [inline Jenkinsfile:42]: The approach to manage exceptions is different than the used in retorch-st-fulteaching:
```
            catchError(buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-setup.sh tjobc 0'
              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-testexecution.sh tjobc 0 https://full-teaching- 5000 "FullTeachingEndToEndRESTTests#attendersRestOperations"'
              sh '$SCRIPTS_FOLDER/tjoblifecycles/tjob-teardown.sh tjobc 0'
            }// EndExecutionStageErrorTJobC
```

File: retorchfiles/scripts/coilifecycles/coi-setup.sh [hunk 2]
```diff
@@ -5,13 +5,13 @@ SEL_LOG_DIR="/opt/selenoid/logs/"
...
-    find "$dir" -mindepth 1 -maxdepth 1 -mtime +$days -exec rm -rf {} \;
-    echo "Removing older files ($days days) in $dir! The number of files after remove:"
-    find "$dir" | wc -l
+  dir=$1
+  days=$2
+  echo "Removing older files ($days days) in $dir! The number of files prior to remove:"
+  find "$dir" | wc -l
+  find "$dir" -mindepth 1 -maxdepth 1 -mtime +$days -exec rm -rf {} \;
+  echo "Removing older files ($days days) in $dir! The number of files after remove:"
+  find "$dir" | wc -l
```
Comment (javiertuya) [inline retorchfiles/scripts/coilifecycles/coi-setup.sh:14]: Remove formatting changes here and the next changed files (see https://github.com/giis-uniovi/.github/blob/main/profile/CONTRIBUTING.md, third item on PR).

**Review by javiertuya — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (augustocristian) [reply Jenkinsfile:42]: I've made the necessary updates in the RETORCH project's base. Once this PR would be accepted I'll also ensure that the changes are reflected in https://github.com/giis-uniovi/retorch-st-fullteaching/pull/80."

**Comment by augustocristian** [comment]
While addressing the comments @javiertuya , I noticed that certain test was failing because of the system's fluctuating speed/performance. This issue has been resolved in https://github.com/giis-uniovi/retorch-st-eShopContainers/pull/52.

**Comment by javiertuya** [comment]
@augustocristian Note that you updated the PR, but the build status is unstable

**Comment by javiertuya** [comment]
@augustocristian Another comment: don't force push after the review has started: https://github.com/giis-uniovi/.github/blob/main/profile/CONTRIBUTING.md, fourth item on PRs.

File: Jenkinsfile (hunk 1 above)
Comment (augustocristian) [reply Jenkinsfile]: Yes, this was redundant. Initially, I believed that 'catcherror' only specified the exception behavior of the step. However, it has now been correctly implemented

**Comment by augustocristian** [comment]
This PR requires the  changes included in https://github.com/giis-uniovi/retorch-st-eShopContainers/pull/52 to correctly pass the test that is failing

**Pull request merged**